### PR TITLE
fix: add patch for updating compiler header for redis-5.0.5 in Ubuntu 22

### PR DESCRIPTION
**Pull request by rishavOneForAll**

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -9,6 +9,13 @@
   tags:
     - install
 
+- name: "Add patch for redis-5.0.5 in Ubuntu 22.04"
+  ansible.builtin.lineinfile:
+    path: /usr/local/src/redis-5.0.5/src/sds.h
+    regexp: '^const char \*SDS_NOINIT;'
+    line: 'extern const char *SDS_NOINIT;'
+  when: redis_version == '5.0.5'
+
```
Comment (imrenagi) [inline tasks/main.yml:18]: so for this following changes, we only need to bump role version we use later on in the redis installation playbook, right?

**Comment by imrenagi** [comment]
@rishavOneForAll can you please help check whether there is similar fix from the upstream repository? 

If possible, can you please also check the latest changes from the upstream and pull rebase it to the master branch later? Thanks!

File: tasks/main.yml (hunk 1 above)
Comment (rishavOneForAll) [reply tasks/main.yml:18]: yes, correct

**Review by imrenagi — APPROVED** [review]
Approved. please help updating redis workflow playbook with this version ya. thanks!

File: tasks/main.yml [hunk 2]
```diff
@@ -9,6 +9,13 @@
   tags:
     - install
 
+- name: "Add patch for redis-5.0.5 in Ubuntu 22.04"
+  ansible.builtin.lineinfile:
+    path: /usr/local/src/redis-5.0.5/src/sds.h
+    regexp: '^const char \*SDS_NOINIT;'
+    line: 'extern const char *SDS_NOINIT;'
+  when: redis_version == '5.0.5'
```
Comment (rizkyekoputra) [inline tasks/main.yml:17]: In the task title this task specific for ubuntu 22, dont we need to put that condition here also @rishavOneForAll?

**Pull request merged**